doc/callb.on_aborted.md

```diff
@@ -0,0 +1,10 @@
+## on_aborted;
+
+Inform application that the remote end has aborted the flow.
+
+Conforms to the ABORT event in D1.2 section 3.2.4.
+
+**Availability:** Currently only available when NEAT selects SCTP
+  transport. The specification in D1.2 and
+  draft-ietf-taps-transports-usage-00 does not specify this event for
+  TCP; we are investigating how we might extend this work for TCP.
```

Comment:
Totally agree. The spec says it is undefined for TCP, but I plan to implement something for this in the near future - would like to keep it out of this PR.